lambda/src/main/resources/db/migration/V94__update_and_insert_displayproperties_data.sql

```diff
@@ -0,0 +1,14 @@
+-- Update DisplayProperties table
+
+UPDATE "DisplayProperties" SET "Value" = '3' WHERE "PropertyName" = 'DescriptionAlternate' and "Attribute" = 'Group';
+UPDATE "DisplayProperties" SET "Value" = '3' WHERE "PropertyName" = 'TitleAlternate' and "Attribute" = 'Group';
+UPDATE "DisplayProperties" SET "Value" = 'small text' WHERE "PropertyName" = 'DescriptionAlternate' and "Attribute" = 'ComponentType';
+
+-- Insert new row to DisplayProperties table
+
+INSERT INTO "DisplayProperties" ("PropertyName","Attribute","Value","AttributeType")
+VALUES
+    ('ClosurePeriod','ComponentType','small text','integer'),
```

Comment:
I think `AttributeType` should be `'text'` for `ComponentType` attribute

Reply:
Correct me if I'm wrong but since we create a `TextField` for both "ClosurePeriod" and "TitleAlternative/DescriptionAlternative" there is a `datatype` attribute that specifies whether it is a `numeric.` or `text` which is why I set the "ClosurePeriod" to be a ``integer`. `ClosurePeriod` only takes an integer from what I can see.

Reply:
Actually, `AttributeType` is about only the attribute value. Here, `'small text'` (value) is a string, so, it should be `text`.